<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheMainForm</class>
 <widget class="QMainWindow" name="TheMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TheMainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_35">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_34">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_33">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>CAN卡</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QPushButton" name="pushButton_Start_can">
                     <property name="text">
                      <string>启动can卡</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>ID</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="comboBox_can_id">
                       <property name="currentIndex">
                        <number>3</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>ID_A</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>ID_B</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>配置</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>NULL</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>UDP</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QPushButton" name="pushButton_start_udp">
                     <property name="text">
                      <string>启动UDP</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>ID</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="comboBox_udp_id">
                       <property name="currentIndex">
                        <number>3</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>ID_A</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>ID_B</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>配置</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>NULL</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>ID</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>USER_CAN_ID_CONFIG_RESPONSE: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>USER_CAN_ID_CONFIG_REQUEST:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>USER_CAN_ID_B_RESPONSE: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>USER_CAN_ID_B_REQUEST: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>USER_CAN_ID_A_RESPONSE: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>USER_CAN_ID_A_REQUEST: </string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_config_response">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_config_request">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_B_response">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_B_request">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_A_response">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_A_request">
                     <property name="text">
                      <string>NULL</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>调试</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_32">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_31">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_27">
                     <item>
                      <widget class="QCheckBox" name="checkBox_oneDriver">
                       <property name="text">
                        <string>单机调速</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_26">
                     <item>
                      <widget class="QComboBox" name="comboBox_A_B">
                       <item>
                        <property name="text">
                         <string>电机A</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>电机B</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_vate"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_25">
                   <item>
                    <widget class="QPushButton" name="pushButton_send_vate">
                     <property name="text">
                      <string>发送</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_25">
                     <property name="text">
                      <string>r/min</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>28</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_30">
                   <item>
                    <widget class="QCheckBox" name="checkBox_doubleDriver">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>双机调速</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_A">
                     <property name="text">
                      <string>左转</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>88</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_29">
                   <item>
                    <widget class="QPushButton" name="pushButton_W">
                     <property name="text">
                      <string>前进</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_Stop">
                     <property name="text">
                      <string>停止</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_S">
                     <property name="text">
                      <string>后退</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_28">
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>88</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_D">
                     <property name="text">
                      <string>右转</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>88</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QPushButton" name="pushButton_close">
            <property name="text">
             <string>close</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>568</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
